api/elastic.py

```diff
@@ -227,6 +233,65 @@ def titleQuery(self, titleText):
                 )
             )
         ])
+    
+    #Query for the identifier value
+    def identifierQuery(self, identifierText):
+        self.searchedFields.extend(['identifiers', 'editions.identifiers'])
+
+        return Q('bool', should=[
+            Q(
+                'query_string',
+                query=identifierText,
+                fields=['identifiers.*'],
+                default_operator='and'
+            ),
+            Q(
+                'nested',
+                path='editions',
+                query=Q(
+                    'query_string',
+                    query=identifierText,
+                    fields=['editions.identifiers.*'],
+                    default_operator='and'
+                )
+            )
+        ])
+
+    #Query for the identifier authority followed up with the identifier value
+    def authIdentQuery(self, authorityList, authIdentText):
+        self.searchedFields.extend(['identifiers', 'editions.identifiers'])
+
+        authIdentList = authIdentText.split(': ')
+        authority = authIdentList[0]
+        identifier = authIdentList[1]
+
```

Comment:
For updating this I think we can look at the length of this array. If it's less than 2 we run the query from `identifierQuery` if it's equal to 2 then we run the query below. That's it!